### DISCUSSION TITLE
No media control/info

### DISCUSSION THREAD

**Discussion by bkbilly** [timmo001/system-bridge#364] [General]
# Description
I am trying to get info about the current playing media but I get an empty reply.
I've tried it on both Ubuntu 20.04 and Windows 10 and they behave exactly the same way.

## How to reproduce the issue (if applicable)
I am using Postman to send request to my PC using a get request to http://localhost:9170/media
I've also tried to send a PUT request to /media/playpause, but I get `successful: false`

I have to mention that I've tried getting the request while playing music on VLC and on Chrome Youtube.

## Screenshots (if applicable)
![image](https://user-images.githubusercontent.com/518494/120542794-32674f80-c3f4-11eb-846a-c9975aea6bcb.png)

**Comment by timmo001** [comment]
This will be for the internal media player which can be used by posting to this endpoint. If there is no media player active (from the app) the get request will return nothing.

System media controls are something I want to add also, but could not find the right library or documentation to interact with the system media controls (cross-platform)
